index_en.html

```diff
@@ -318,6 +335,32 @@ <h2>Interactive wall</h2>
         In this kind of scene, the Bullet Chatting is also displayed separately, not attached to the video.
       </p>
     </section>
+    <section>
+      <h2>Masking</h2>
+      <p>
+        <p>
+          When watching a video, there may be a large number of bullet chatting appearing on the screen at the same time, causing the content of the video itself to be covered, which will affect watching the media content. For example, when the user is learning to dance, they may not see the movement of the dancer. [To be translated part: 这种情况下，就可以借助计算机视觉和AI的能力，对视频内容进行分析，并将之前已经定义好的“视频主体内容”进行识别，生成蒙版并分发给客户端后，客户端利用CSS3的特性进行渲染从而达成弹幕不会覆盖定义的“视频主体内容”的效果，这种弹幕被称为“蒙版弹幕”，也被称为“防挡弹幕”。]
```

Comment:
s/causing the content of the video itself to be covered, which will affect watching the media content/which may unexpectedly cover the video itself, and disrupt users from watching the media content

Reply:
Fixed: https://github.com/w3c/danmaku/pull/14/commits/def550f751355c4c16ffc31307e2d245f3a23788